### ISSUE TITLE
Bundled macos-release module missing Monterey

### ISSUE THREAD

**Issue by cmsj** [Gottox/irc-message-action#11]
Hi!

I'm testing the beta version of GitHub's macOS-12 runner and I'm hitting a weird traceback I can't figure out, with irc-message-action. It's entirely likely that this is an issue with github's new runner environment, but I wanted to mention it here in case this is a subtle incompatibility with newer things. If it helps, the various versions in the macOS-12 runner are listed [here](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-12-Readme.md)

```
Run Gottox/irc-message-action@v2.1.0
  with:
    channel: #hammerspoon
    nickname: HSGHA
    message: CI for https://github.com/Hammerspoon/hammerspoon/compare/518187c230f7...6eda506cd553 (Chris Jones):
    * Update Xcode version too
    * 	 Executed 302 tests, with 50 tests skipped and 2 failures (0 unexpected) in 672.422 (672.791) seconds
  
    server: irc.libera.chat
    port: 6697
    notice: false
    tls: true
    response_timeout: 60
    excess_flood: throttle
    debug: false
  env:
    IS_CI: 1

/Users/runner/work/_actions/Gottox/irc-message-action/v2.1.0/node_modules/@octokit/rest/node_modules/universal-user-agent/dist-node/index.js:17
    throw error;
    ^

TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
    at macosRelease (/Users/runner/work/_actions/Gottox/irc-message-action/v2.1.0/node_modules/macos-release/index.js:26:26)
    at osName (/Users/runner/work/_actions/Gottox/irc-message-action/v2.1.0/node_modules/os-name/index.js:21:18)
    at getUserAgent (/Users/runner/work/_actions/Gottox/irc-message-action/v2.1.0/node_modules/@octokit/rest/node_modules/universal-user-agent/dist-node/index.js:11:53)
    at parseOptions (/Users/runner/work/_actions/Gottox/irc-message-action/v2.1.0/node_modules/@octokit/rest/lib/parse-client-options.js:80:57)
    at new Octokit (/Users/runner/work/_actions/Gottox/irc-message-action/v2.1.0/node_modules/@octokit/rest/lib/constructor.js:23:31)
    at Object.<anonymous> (/Users/runner/work/_actions/Gottox/irc-message-action/v2.1.0/node_modules/@actions/github/lib/github.js:16:28)
    at Module._compile (internal/modules/cjs/loader.js:999:[30](https://github.com/Hammerspoon/hammerspoon/runs/5760587740?check_suite_focus=true#step:12:30))
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:[32](https://github.com/Hammerspoon/hammerspoon/runs/5760587740?check_suite_focus=true#step:12:32))
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
```

**Comment by cmsj** [comment]
I think I've figured it out, looks like the `macos-release` in your `node_modules` is out of date, because it doesn't include Monterey: https://github.com/Gottox/irc-message-action/blob/main/node_modules/macos-release/index.js

(vs the upstream version, which does: https://github.com/sindresorhus/macos-release/blob/main/index.js )

**Comment by cmsj** [comment]
@Gottox any chance of getting an updated build released? 😁

**Comment by Gottox** [comment]
Fixed in main. Could you recheck?

**Comment by cmsj** [comment]
@Gottox works perfectly, thanks!

**Comment by Gottox** [comment]
fixed in [v2.1.2](https://github.com/Gottox/irc-message-action/releases/tag/v2.1.2)
